macro/src/lib.rs

```diff
@@ -33,25 +33,53 @@ struct AttrArgs {
     key_expr: Expr,
     store_type: Option<Type>,
     store_init: Option<Expr>,
+    result: Option<bool>,
```

Comment:
```suggestion
    result: bool,
```
> Booleans can be used without assiging a value. as a flag.
   If omitted they are set to false